src/views/CharacterListView.js

```diff
@@ -11,11 +12,14 @@ class CharacterListView extends React.Component {
 
   componentDidMount() {
     // call our action
+    console.log('character list view says:', this.props )
+    this.props.addCharacter('Jtonna');
```

Comment:
What did you intend to do with the string argument you passed here? It didn't seem to be used in the action creator.